### PR TITLE
Disnix update

### PR DESCRIPTION
**Pull request by svanderburg**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [X] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 121823` at 14971b1a run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li><a href="https://gist.github.com/ef5fa24d270c9cca4a30f9a6689da1b3">dydisnix</a></li>
  </ul>
</details>
<details>
  <summary>4 packages built successfully:</summary>
  <ul>
    <li>DisnixWebService</li>
    <li>disnix</li>
    <li>disnixos</li>
    <li>dysnomia</li>
  </ul>
</details>

Note that build failures [may predate](https://github.com/nix-community/hydra-check) this PR, and could be nondeterministic or hardware dependent.
Please exercise your independent judgement.

File: pkgs/tools/package-management/disnix/dydisnix/default.nix [hunk 1]
```diff
@@ -1,18 +1,18 @@
-{ lib, stdenv, fetchFromGitHub, autoconf, automake, libtool , pkg-config, glib, libxml2, libxslt, getopt, libiconv, gettext, nix, disnix, libnixxml }:
+{ lib, stdenv, fetchFromGitHub, autoconf, automake, libtool , pkg-config, glib, libxml2, libxslt, getopt, libiconv, gettext, nix, disnix }:
 
 stdenv.mkDerivation rec {
-  version="2020-07-04";
+  version="2020-11-02";
```
Comment (SuperSandro2000) [inline pkgs/tools/package-management/disnix/dydisnix/default.nix:4]: The version number should be prefixed with ``unstable-`` and also the file could get some formatting with nixpkgs-fmt.

File: pkgs/tools/package-management/disnix/dysnomia/default.nix [hunk 2]
```diff
@@ -1,5 +1,5 @@
 { lib, stdenv, fetchurl, netcat
-, systemd ? null, ejabberd ? null, mysql ? null, postgresql ? null, subversion ? null, mongodb ? null, mongodb-tools ? null, influxdb ? null, supervisor ? null, docker ? null
+, systemd ? null, ejabberd ? null, mysql ? null, postgresql ? null, subversion ? null, mongodb ? null, mongodb-tools ? null, influxdb ? null, supervisor ? null, docker ? null, nginx ? null, s6-rc ? null, xinetd ? null
```
Comment (SuperSandro2000) [inline pkgs/tools/package-management/disnix/dysnomia/default.nix:2]: It wouldn't hurt to add a line break here.

Comment (svanderburg) [reply pkgs/tools/package-management/disnix/dysnomia/default.nix:2]: yes maybe. I'll see if I can change that

File: pkgs/tools/package-management/disnix/dydisnix/default.nix (hunk 1 above)
Comment (svanderburg) [reply pkgs/tools/package-management/disnix/dydisnix/default.nix:4]: I didn't know that this is a requirement. I simply just changed what was already there.

I happen to be the author of dydisnix, but I'm not the maintainer of the package. It is @tomberek who packaged it and apparently has a use for it. I only use in experimental setups and typically deploy it directly from the Nix expressions in the development repository.

Comment (svanderburg) [reply pkgs/tools/package-management/disnix/dydisnix/default.nix:4]: I can of course tidy up the expression, no problem :)

**Pull request merged**